### PR TITLE
Add instructions about how to release actions

### PR DESCRIPTION
**Pull request by scheibinger**
I didn't know how to release and forgot and how changeset works, so I figured out that would be nice to have short instructions here.

Do we have a general doc about changeset I could refer here?

### CODE REVIEW

**Review by njegosrailic — APPROVED** [review]

**Review by njegosrailic — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -147,6 +147,27 @@ the minimum required files.
 Once ready to release this new action, run `pnpm changeset` to create the
 initial changeset file and bump the minor version.
 
+### Releasing updated actions
+
+Let's assume that you made a PR with a change in one of the reusable actions,
+and you would like to release it. To do that, follow the steps below:
```
Comment (chainchad) [inline README.md:153]: We should reword this so it's obvious that ideally you'll run `pnpm changeset` before creating the PR. The changeset should be in the branch before the PR is merged.

File: README.md [hunk 2]
```diff
@@ -147,6 +147,27 @@ the minimum required files.
...
+- Run `pnpm changset` in the git top level directory.
+- This repo contains multiple packages, so it will ask you for which package it
+  should generate changeset update.
+- Answer remaining questions. At the end, you will have a new
+  `.changeset/<random-name>.md` file generated.
+- Now you need to commit and push changeset updates to your PR
+
+After merging your PR, the renovate bot will create a PR like
+[this one](https://github.com/smartcontractkit/.github/pull/540) Which contains
+a release bump.
```
Comment (chainchad) [inline README.md:164]:
```suggestion
After merging your PR, a changesets CI job will create a PR like
[this one](https://github.com/smartcontractkit/.github/pull/540) which contains
a release bump.
```

**Review by chainchad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scheibinger) [reply README.md]: It's updated now

File: README.md (hunk 2 above)
Comment (scheibinger) [reply README.md]: fixed

**Review by chainchad — APPROVED** [review]

**Pull request merged**